src/common/adxl345.rs

```diff
@@ -6,11 +6,14 @@ use crate::google;
 
 use super::board::Board;
 use super::config::ConfigType;
+use super::generic::DoCommand;
 use super::i2c::I2CHandle;
 use super::movement_sensor::MovementSensorType;
 use super::registry::{get_board_from_dependencies, ComponentRegistry, Dependency};
 use super::status::Status;
 
+use macros::DefaultDoCommand;
```

Comment:
just DoCommand

Reply:
never mind, macro and trait usages never intersect so the compiler apparently infers appropriately